lib/leche.js

```diff
@@ -142,6 +142,12 @@ module.exports = {
 
 		var fake = createObject(template);
 
+		function fakeFn(name) {
```

Comment:
Rather than change functionality around, I'd prefer to just disable the ESLint rule around the loop. That can be done with `/*eslint-disable no-loop-func*/` before and `/*eslint-enable no-loop-func*/` after.